localstack/services/cloudformation/cloudformation_starter.py

```diff
@@ -575,6 +575,26 @@ def SNS_Topic_get_cfn_attribute(self, attribute_name):
     SNS_Topic_get_cfn_attribute_orig = sns_models.Topic.get_cfn_attribute
     sns_models.Topic.get_cfn_attribute = SNS_Topic_get_cfn_attribute
 
+    # Patch ES get_cfn_attribute(..) method
+    def ES_get_cfn_attribute(self, attribute_name):
+        if attribute_name in ['Arn', 'DomainArn']:
+            return aws_stack.es_domain_arn(self.params.get('DomainName'))
+        if attribute_name == 'DomainEndpoint':
+            es_details = aws_stack.connect_to_service('es').describe_elasticsearch_domain(
```

Comment:
nit: Can we try to cache this value, to improve the performance and avoid that it gets requested from the API multiple times? Something like:
```
        if attribute_name == 'DomainEndpoint':
            if not hasattr(self, '_domain_endpoint'):
                es_details = aws_stack.connect_to_service('es').describe_elasticsearch_domain(
                    DomainName=self.params.get('DomainName'))
                self._domain_endpoint = es_details['DomainStatus']['Endpoint']
            return self._domain_endpoint
        ...
```